index.html

```diff
@@ -209,6 +209,16 @@ <h3 id="server-performance"><a href="#server-performance">Server performance</a>
             <td class="alert">no</td>
           </tr>
           <tr>
+            <td><a href="https://iojs.org/api/https.html">io.js</a></td>
+            <td class="ok">yes</td>
+            <td class="ok"><a href="https://github.com/joyent/node/pull/5882">yes</a></td>
+            <td class="ok"><a href="https://github.com/joyent/node/pull/7469">yes</a></td>
```

Comment:
@indutny thanks, just to confirm that I'm following the logic in [1].. io.js provides a callback that's triggered when client sets the OCSP flag. From there the server is responsible for implementing own logic for querying the OSCP server/fetching OCSP response from cache, etc. Correct?

[1] https://iojs.org/api/tls.html#tls_event_ocsprequest